internal/operands/node-labeller/reconcile.go

```diff
@@ -109,7 +104,7 @@ func reconcileConfigMap(request *common.Request) error {
 		})
 }
 
-func reconcileDaemonSet(request *common.Request) error {
+func reconcileDaemonSet(request *common.Request) (common.ResourceStatus, error) {
 	nodeLabellerSpec := &request.Instance.Spec.NodeLabeller
 	daemonSet := newDaemonSet(request.Namespace)
 	addPlacementFields(daemonSet, &nodeLabellerSpec.Placement)
```

Comment:
This function should return `common.CreateOrUpdateResourceWithStatus()` because `DaemonSet`s support conditions and deploy pods

Reply:
I missed this. Updated.